src/js/components/Lotto.js

```diff
@@ -1,36 +1,82 @@
-import { LottoPurchase } from "./LottoPurchase.js";
-import { LottoTickets } from "./LottoTickets.js";
-//import { LottoWinning } from "./LottoWinning.js";
+import LottoReward from "../domain/LottoReward.js";
+import LottoShop from "../domain/LottoShop.js";
+import LottoTickets from "../domain/LottoTickets.js";
+import LottoModal from "./LottoModal.js";
+import LottoPurchaseForm from "./LottoPurchaseForm.js";
+import LottoTicketsForm from "./LottoTicketsForm.js";
+import WinningLottoForm from "./WinningLottoForm.js";
 
 export default class Lotto {
-    lottoTickets = null;
-    lottoPurchase = null;
-    $lottoPurchaseArea = null;
-    $lottoAmountArea = null;
-    $lottoTicketArea = null;
-    //$lottiWinningArea = null;
+    price;
+    lottos;
+    winningNumbers;
+    bonusNumber;
+    rating;
+    rate;
 
     constructor() {
-        this.$lottoPurchaseArea = document.querySelector("#lotto-purchase-area");
-        this.$lottoAmountArea = document.querySelector("#lotto-amount-area");
-        this.$lottoTicketArea = document.querySelector("#lotto-ticket-area");
-        //$lottiWinningArea = document.querySelector("#lotto-winning-area");
-
-        this.lottoPurchase = new LottoPurchase(this.$lottoPurchaseArea, {
-            onLottoPurchase: (tickets) => {
-                this.lottoTickets.setTickets(tickets);
-                this.lottoTickets.setAmount(tickets.length);
-                this.lottoTickets.render();
-                this.lottoTickets.mounted();
-                this.lottoTickets.setEvent();
-
-                // this.lottoWinning.render();
-                // this.lottoWinning.mounted();
-                // this.lottoWinning.setEvent();
-            },
+        this.lottoPurchaseForm = new LottoPurchaseForm({
+            onPurchase: (price) => this.onPurchase(price),
         });
+        this.lottoTicketsForm = new LottoTicketsForm();
+        this.lottoTickets = new LottoTickets();
+        this.winningLottoForm = new WinningLottoForm({
+            onReward: (winningNumber, bonusNumber) => this.onReward(winningNumber, bonusNumber),
+        });
+        this.lottoModal = new LottoModal({
+            onReset: () => this.onReset(),
+        });
```

Comment:
onReward, onReset 등은 arrow function으로 작성해주면 
```suggestion
        const { onReward, onReset } = this;
        this.winningLottoForm = new WinningLottoForm({ onReward });
        this.lottoModal = new LottoModal({ onReset });
```
이렇게 사용할 수 있겠네요!
그리고 이럴 땐 `on` 이라는 keyword는 제거하는게 적합해보여요 ㅎㅎ
단순히 동사 형태로 사용해야 함수로서의 기능이 돋보일 것 같습니다.